content/20.treatments.md

```diff
@@ -846,6 +846,23 @@ They may also engage the host immune response, encouraging the immune system to
 Given the cytokine storm that results from immune system activation in response to the virus, which has been implicated in worsening of the disease, a neutralizing antibody (nAb) may be preferable.
 Upcoming work may explore the specificity of nAbs for their target, mechanisms by which the nAbs impede the virus, and improvements to antibody structure that may enhance the ability of the antibody to block viral activity.
 
+There is also some research into possible potential therapeutics or prophylactics which interact with components of the innate immune response.
+Broadly, there are a variety of toll-like receptors which are examples of pattern recognition receptors (PRRs), innate immune response components which recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
+Note that Toll-like receptors form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses.
+Due to the relationship between coronaviral infections, ARDS, and the toll-like receptors TLR3 and TLR4, TLR agonists and antagonists hold some potential for broad-spectrum prophylaxis.
+In mouse models, poly(I:C) and lipopolysaccharide, which are agonists of TLR3 and TLR4, respectively, showed protective effects when administered prior to SARS-CoV exposure [@doi:10.1128/JVI.01410-12].
+Another type of approach which is being investigated is the possible use of what is termed trained immunity, in particular as elicited by non-SARS-CoV-2 whole-microorganism vaccines (or other microbial stimuli), as a tool which might be shown to generate heterologous protective effects with respect to SARS-CoV-2 susceptibility or severity [@doi:10.1016/j.cell.2020.04.042].
+In a recent review [@doi:10.1038/s41577-020-0285-6], trained immunity was defined as forms of memory which are temporary (e.g., months or years, and reversible), displayed by innate immune cells and innate immune features of other cells, displayed as increased responsiveness to future same or heterologous pathogen infection or sometimes decreased responsiveness or immunological tolerance, and established through epigenetic and metabolic mechanisms; the review also notes various ways in which trained immunity may be related to pathology, inflammatory-related diseases, conditions and immunology associated with aging, and cancer.
+One type of stimulus which research indicates can induce trained immunity is BCG vaccination (BCG is an acronym for bacillus Calmette-Guerin).
+BCG is a live attenuated strain derived from an isolate of Mycobacterium bovis with the common reason for application being prevention of tuberculosis in humans.
+With respect to SARS-CoV-2 specifically, clinical trials in non-SARS-CoV-2-infected adults have been setup to assess the possible efficacy of BCG vaccination.
```

Comment:
I chose the vague "efficacy" b/c I meant 2 ways in which it might be effective: (1) reduce susceptibility / prevent infection / ~ prophylaxis and (2) reduce disease severity / partial therapeutic. So, if you think it's better, we could write "assess BCG vaccination as a potential prophylactic and/or partial therapeutic." ??
(Note, I already mentioned in an earlier line above the other possible wording which is closer to what I saw in article "... might be shown to generate heterologous protective effects with respect to SARS-CoV-2 susceptibility or severity".)

Reply:
Ah, if you would be open to the "potential prophylactic and/or partial therapeutic phrasing," I think that would really help people like me who don't know much about this!